main.yml

```diff
@@ -352,8 +350,17 @@ settings:
   title: Cloud Services Settings
   frontend:
     sub_apps:
+      - id: ''
+        default: true
+        title: 'User Preferences'
       - id: hooks
+      - id: sources
+      - id: rbac
   top_level: true
+  isBeta: true
+  deployment_repo: https://github.com/RedHatInsights/settings-frontend-build
+  paths:
+  - /settings
```

Comment:
Let's indent this

Reply:
To clarify, just the `- /settings` line needs to be indented here.

Reply:
We'll need another `- settings/${config}` path, where I don't really know how to name the `config` app. We'll be configurating application in tat section, so `app` would be nice, but that's not really a good application ID. @kruai any ideas?

Reply:
We can call it for example `- settings/sites` so it the whole url will be `settings/sites/insights`. What do you think?

Reply:
Let's wait to see what Kendra's mocks say, but before it was going to be 
`settings/applications/insights`